app/Http/Controllers/Api/SessionController.php

```diff
@@ -34,6 +34,17 @@ public function create(Request $request)
         $checksession = Session::where(['session' => $_session])->first();
         if (empty($checksession)) {
             if ($session->save()) {
+
+                 $is_current = $request->get('is_current');
+
+                 if ($is_current != 0) {                     
+                    $getSessionId = Session::where(['session' => $_session])->first()->id;
```

Comment:
DRY! will $checksession->id do?

Reply:
Could you please explain what this method does from line 36?

Reply:
this should'nt be discussed here, i don't find it neccessary, DM 😃